sections/variant-list.liquid

```diff
@@ -83,6 +88,13 @@
             </tbody>
           </table>
         </div>
+        <noscript>
+          {%- if product.has_only_default_variant or product.variants.size == 1 -%}
```

Comment:
We can. May I know why you think this order better? My logic was If I put { if } outside of <noscript> it would be checked every time with or without JS but now it's checked only when JS is off.